<?xml version="1.0"?>

<launch>
    <rosparam file = "$(find final_challenge)/config/control__file.yaml" command = "load"  />
    <rosparam file = "$(find final_challenge)/config/setPoint_file.yaml" command = "load"  />
	<node pkg="final_challenge" type="open_loop_controller.py" name="controller" output="screen"/>
	<node pkg="final_challenge" type="keyboard_cpp" name="keyboard" output="screen" launch-prefix="gnome-terminal --command" />
	<node pkg="final_challenge" type="set_point.py" name="set_point" output="screen"/>
	<node pkg="rosserial_python" type="serial_node.py" name="serial" output="screen"  args="/dev/ttyACM0" />
	<node pkg="rqt_plot" type="rqt_plot" name="final_plot" args="/motor_output/vel /error/data /set_point/vel"/>
</launch>